src/services/email.service.js

```diff
@@ -0,0 +1,69 @@
+require('dotenv/config');
+
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
```

Comment:
The `port` value in the nodemailer transport configuration should be a number. Ensure that `process.env.SMTP_PORT` is correctly set as a number in your environment variables.